slackbot.sh

```diff
@@ -4,6 +4,7 @@
 # Expects the following environment variables to be available:
 #  * GIT_REPO_URL      The git repository URL (e.g. git@github.com:newsuk/project.git)
 #  * GIT_HASH          The commit hash of the build, must be tagged!
+#  * GIT_TOKEN         The git token for git api
```

Comment:
Can we add this to the README as well?  Perhaps with a little more info about what the token actually is, why its needed and perhaps a link to the GitHub docs about how to create them.

Reply:
Sure